### PR TITLE
K8SPG-597 Add PG operator 2.4.0

### PR DESCRIPTION
**Pull request by ptankov**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.2.4.0.pg-operator.dep.json [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+            },
+            "15.7"
+          ]
+        },
+        {
+          "<":[
+            {
+              "var":"productVersion"
+            },
+            "16.1"
```
Comment (tplavcic) [inline sources/operator.2.4.0.pg-operator.dep.json:26]: I think it should be 16.0 here

File: sources/operator.2.4.0.pg-operator.dep.json [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+            },
+            "14.12"
+          ]
+        },
+        {
+          "<":[
+            {
+              "var":"productVersion"
+            },
+            "15.5"
```
Comment (tplavcic) [inline sources/operator.2.4.0.pg-operator.dep.json:46]: I think it should be 15.0 here

File: sources/operator.2.4.0.pg-operator.dep.json [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+            },
+            "13.15"
+          ]
+        },
+        {
+          "<":[
+            {
+              "var":"productVersion"
+            },
+            "14.10"
```
Comment (tplavcic) [inline sources/operator.2.4.0.pg-operator.dep.json:66]: I think it should be 14.0 here

File: sources/operator.2.4.0.pg-operator.dep.json [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+            },
+            "12.19"
+          ]
+        },
+        {
+          "<":[
+            {
+              "var":"productVersion"
+            },
+            "13.13"
```
Comment (tplavcic) [inline sources/operator.2.4.0.pg-operator.dep.json:86]: 13.0

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by tplavcic — APPROVED** [review]

**Review by fiowro — APPROVED** [review]

**Pull request merged**